src/test/java/io/strimzi/kafka/bridge/http/DefaultTest.java

```diff
@@ -0,0 +1,135 @@
+                        assertTrue(paths.containsKey("/consumers/{groupid}/instances/{name}/assignments"));
+                        assertEquals(HttpOpenApiOperations.ASSIGN.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/consumers/{groupid}/instances/{name}/assignments").getJsonObject("post").getString("operationId"));
+                        assertTrue(paths.containsKey("/consumers/{groupid}/instances/{name}/records"));
+                        assertEquals(HttpOpenApiOperations.POLL.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/consumers/{groupid}/instances/{name}/records").getJsonObject("get").getString("operationId"));
+                        assertTrue(paths.containsKey("/consumers/{groupid}/instances/{name}/offsets"));
+                        assertEquals(HttpOpenApiOperations.COMMIT.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/consumers/{groupid}/instances/{name}/offsets").getJsonObject("post").getString("operationId"));
+                        assertTrue(paths.containsKey("/topics/{topicname}"));
+                        assertEquals(HttpOpenApiOperations.SEND.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/topics/{topicname}").getJsonObject("post").getString("operationId"));
+                        assertTrue(paths.containsKey("/topics/{topicname}/partitions/{partitionid}"));
+                        assertEquals(HttpOpenApiOperations.SEND_TO_PARTITION.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/topics/{topicname}/partitions/{partitionid}").getJsonObject("post").getString("operationId"));
+                        assertTrue(paths.containsKey("/healthy"));
+                        assertEquals(HttpOpenApiOperations.HEALTHY.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/healthy").getJsonObject("get").getString("operationId"));
+                        assertTrue(paths.containsKey("/ready"));
+                        assertEquals(HttpOpenApiOperations.READY.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/ready").getJsonObject("get").getString("operationId"));
+                        assertTrue(paths.containsKey("/openapi"));
+                        assertEquals(HttpOpenApiOperations.OPENAPI.toString(), bridgeResponse.getJsonObject("paths").getJsonObject("/openapi").getJsonObject("get").getString("operationId"));
+                        assertFalse(paths.containsKey("/karel"));
+                        assertEquals(15, bridgeResponse.getJsonObject("definitions").getMap().size());
+                        assertEquals(4, bridgeResponse.getJsonArray("tags").size());
+                    });
+                    context.completeNow();
+                });
+    }
+}
```

Comment:
if you are grouping I would not leave these tests in the base class. Maybe the first two are about "healthy". The openapi one ... I don't have a strong opinion.

Reply:
Sorry @ppatierno, but we can't have tests in base class because of having child classes. Every child class will execute these ST and we will have 4-5 excess-runs for each ST

Reply:
My mistake I was seeing the wrong file. ;)